recipes-sota/lmp-device-register/lmp-device-register_git.bb

```diff
@@ -4,7 +4,7 @@ LIC_FILES_CHKSUM = "file://COPYING.MIT;md5=838c366f69b72c5df05c96dff79b35f2"
 
 DEPENDS = "boost curl ostree glib-2.0"
 
-SRCREV = "579afb7488534acd04d7206ce39422553d62a801"
+SRCREV = "670ab2d040df3413938377816fce8b8933a4be91"
```

Comment:
This doesn't include @doanac latest changes.  Is that ok?

Reply:
Yes, asked that to @doanac earlier today and he wanted to update that on another update.